jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -45,8 +45,8 @@ teardown() {
   export DATASOURCES="RHPAM"
   export RHPAM_DRIVER="mariadb"
   export RHPAM_DATABASE="rhpam-mariadb"
-  export RHPAM_USERNAME="rhpam-user"
-  export RHPAM_PASSWORD="rhpam-pwd"
+  export RHPAM_USERNAME="rhpam-user$0"
+  export RHPAM_PASSWORD="rhpam-pwd$0"
```

Comment:
check these values as well

[ "${EJB_TIMER_USER}" = "rhpam-pwd\$0" ]

Reply:
added 
[ "${EJB_TIMER_PASSWORD}" = "rhpam-pwd$0" ]
and
[ "${EJB_TIMER_USERNAME}" = "rhpam-user$0" ]